dpnp/dpnp_algo/dpnp_algo_arraycreation.pyx

```diff
@@ -54,6 +54,9 @@ __all__ += [
 ]
 
 
+ctypedef void(*custom_1in_1out_func_ptr_t)(void * , void * , const int , size_t * , size_t * , const size_t, const size_t)
```

Comment:
1in 1out func? seems it doesn't matches with function signature. Bad naming practice.